vulnerabilities/fixtures/openssl.json

```diff
@@ -40764,5 +40764,19 @@
                 ]
             }
         }
+    },
+    {
+        "model": "vulnerabilities.importer",
+        "pk": 17,
+        "fields": {
+            "name": "xen",
+            "license": "",
```

Comment:
Do you know what the license would be?

Reply:
GPLv2

Reply:
@Pushpit07

> re: GPLv2

if this is it, then state it. Do you have a link to the license info?

Reply:
https://wiki.xenproject.org/wiki/Xen_FAQ_General